xbmc/guilib/GUIControlFactory.h

```diff
@@ -128,6 +128,14 @@ class CGUIControlFactory
   static bool GetScroller(const TiXmlNode* pControlNode,
                           const std::string& scrollerTag,
                           CScroller& scroller);
+  static bool GetPrimitive(const TiXmlElement* node,
+                           const int parentID,
```

Comment:
One last thing, you forgot to update the header :slightly_smiling_face: 
```suggestion
                           int parentID,
```

Reply:
Oh yeah.